Tw quiz

Thomaswaters05 opened the pull request:
Finished.. I needed to add " " instead of "". Ready for review.

PREV POST: (Currently cannot figure out the math for the valid form. I can get the characters to appear + 1 and grow. But i cannot get the tree to follow that path to create that shape. Still working on this but wanted to PR so you could see where I was sitting with my quiz).

Thomaswaters05 committed:
almost there, creating the correct spacing of tree/math equation

Thomaswaters05 committed:
cant figure out equation to make the tree even and validate properly

Thomaswaters05 committed:
finished

copypastedeveloper commented on index.html:
```diff
@@ -0,0 +1,24 @@
+  <title>JS QUIZ</title>
+</head>
+<body>
+
```
INDENTATION IS IMPORTANT!!!!!!

copypastedeveloper commented on quiz.js:
```diff
@@ -0,0 +1,56 @@
+// TW PERSONAL NOTE: Creating the object and deleted my prior wrong doing of height * charoption (<<<--not correct). Create an object, console.log it, and create the next function.
+// think of it as information traveling from one block to the next!! (TW-Study more on this)
+
+function generateTreeObject (height, characterOption) {
```
Can you eliminate this function?  As a rule of thumb, if your function is just calling another function, it's probably extraneous.

copypastedeveloper commented on quiz.js:
```diff
@@ -0,0 +1,56 @@
+    var numSpace = tree.ht-i;
+    console.log(" ".repeat(numSpace) + characters.repeat(numChar));
+  }
+  }
```
INDENTATION IS IMPORTANT!!!!!!!!

copypastedeveloper commented on quiz.js:
```diff
@@ -0,0 +1,56 @@
+
+function correctInput () {
+  if (height.value >= 100) {
+    alert("Please enter value lower than 100. You may choose any character you please for the next input field.");
```
What if the user doesn't select a character?

copypastedeveloper commented:
@Thomaswaters05 a couple of things:

## Requirements
- [x] Fix the indentation issues in the html and js files.  
- [x] Add validation for missing character selection

Everything else looks very nice.

Thomaswaters05 committed:
fixed errors

Thomaswaters05 commented:
Nathan: Ready for re-check please.

Thank you sir! I have fixed the indentation issues and added a validation for both the missing numerical selection and character selection.

copypastedeveloper commented:
@Thomaswaters05 looks good. 👍

Thomaswaters05 commented:
Thank you!!
